src/interactive-cli.ts

```diff
@@ -24,7 +24,7 @@ export async function interactiveCli(baseargv: string[]) {
         if (walletProperties.wallet.includes("Private Key") && walletProperties.path && walletProperties.network) {
             const args = [...baseargv.slice(0, 2), "info", taskConstants[task], `--env-path=${walletProperties.path}`, `--network=${walletProperties.network}`, "--get-hacked"]
             await program.parseAsync(args)
-        } else if (walletProperties.wallet.includes("Public Key")) {
+        } else if (walletProperties.wallet.includes("Public Key") || walletProperties.wallet.includes("Ledger")) {
```

Comment:
create const for "Public Key" and "Ledger" it will be easier for you to maintain and change if required